src/libs/deprecatedAPI.js

```diff
@@ -456,21 +456,6 @@ function User_GetBetas() {
     return Network.post('User_GetBetas');
 }
 
-/**
- * @param {Object} parameters
- * @param {String} parameters.email
- * @param {Boolean} [parameters.requireCertainty]
- * @returns {Promise}
- */
-function User_IsFromPublicDomain(parameters) {
-    const commandName = 'User_IsFromPublicDomain';
-    requireParameters(['email'], parameters, commandName);
-    return Network.post(commandName, {
-        ...{requireCertainty: true},
```

Comment:
We changed this to false in the Web-E PR since all use cases in NewDot default to false in case we don't have a Clearbit response in time.